Graphs/prim.cpp

```diff
@@ -0,0 +1,74 @@
+// Prim's Algorithm is used to find the Minimum Spanning Tree (MST) of a graph, 
+// minimizing the total edge weight while connecting all vertices without cycles.
+
+#include <iostream>
+#include <vector>
+#include <queue>
+#include <utility>
+#include <functional>
```

Comment:
Delete the includes. In the book, we always recommend for the competitor to include only `bits/stdc++.h`, since it includes everything needed and thus, we don't show any includes in any of the code snippets.